go/chat/storage/storage_blockengine.go

```diff
@@ -12,7 +12,7 @@ import (
 	"golang.org/x/net/context"
 )
 
-const blockIndexVersion = 6
+const blockIndexVersion = 7
```

Comment:
I don't think leveldb has support for that. We could build a layer in the future.